test/fixtures/basic/plugins/dependsOnPlugin.ts

```
@@ -0,0 +1,8 @@
+export default defineNuxtPlugin({
+  name: 'depends-on-plugin',
+  dependsOn: ['async-plugin'],
+  async setup () {
+    await new Promise(resolve => setTimeout(resolve, 100))
```

Should we throw an error if `$asyncPlugin` hasn't been injected?